### DISCUSSION TITLE
[Feature]: Enable the feature to manage any DNS Record to compete Vs Plesk

### DISCUSSION THREAD

**Discussion by miguelgargallo** [coollabsio/coolify#3359] [Feature Requests]
### Description

The tool seems not to be able to manage DNS records

### Minimal Reproduction (if possible, example repository)

I cannot find where it is, but it can do it

### Exception or Error

I cannot find where it is, but it can do it

### Version

v4.0.0-beta.146

**Comment by AlejandroAkbal** [comment]
I think this might be out of scope for this project, but would be lovely to see it in the future

**Comment by blinkinglight** [comment]
maybe coredns ? syntax similar to caddy

**Comment by Shadowfita** [comment]
What is your goal with this feature? Wild carded domains with the right CNAME is almost like automatic domain provisioning based on a service.
